wgc/wgc_api.py

```diff
@@ -229,7 +238,10 @@ def get_account_realm(self) -> str:
     #
 
     def auth_server_uri(self) -> str:
-        return 'http://%s:%s/login' % (self.LOCALSERVER_HOST, self.LOCALSERVER_PORT)
+        if platform.system() == 'Windows':
+            return 'http://%s:%s/login' % (self.LOCALSERVER_HOST, self.LOCALSERVER_PORT)
+        
+        return 'http://%s:%s/unsupported_platform' % (self.LOCALSERVER_HOST, self.LOCALSERVER_PORT)
```

Comment:
Probably does not matter, as macOS build is still crashing on start before auth window. I has a few problems with dealing with local files (like creating WCG instance in plugin init, should probably should be skipped until no real support for Mac)

Reply:
@Mixaill